src/main/java/com/gluonhq/utils/MavenArtifactResolver.java

```diff
@@ -97,8 +103,34 @@ private MavenArtifactResolver(List<Repository> repositories) {
         systemSession = createRepositorySystemSession(repositorySystem, DEFAULT_LOCAL_REPO);
         remoteRepositories = new LinkedList<>();
         repositories.forEach(r -> {
+            org.apache.maven.model.RepositoryPolicy releases = r.getReleases();
+            RepositoryPolicy releasesRepositoryPolicy = null;
+            if (releases != null) {
+                releasesRepositoryPolicy = new RepositoryPolicy(releases.isEnabled(),
+                        releases.getUpdatePolicy() == null ? UPDATE_POLICY_NEVER : releases.getUpdatePolicy(),
+                        releases.getChecksumPolicy());
+            }
+            RepositoryPolicy snapshotsRepositoryPolicy = null;
+            org.apache.maven.model.RepositoryPolicy snapshots = r.getSnapshots();
+            if (snapshots != null) {
+                snapshotsRepositoryPolicy = new RepositoryPolicy(snapshots.isEnabled(),
+                        snapshots.getUpdatePolicy() == null ? UPDATE_POLICY_NEVER : snapshots.getUpdatePolicy(),
+                        snapshots.getChecksumPolicy());
+            }
+            if (releasesRepositoryPolicy == null && snapshotsRepositoryPolicy == null) {
+                if (r.getUrl().toLowerCase(Locale.ROOT).contains("/releases")) {
```

Comment:
Maybe test for contains of `/release` and `/snapshot` instead to include more use cases?

Reply:
done